forms/common-themes/src/main/resources/theme/base/account/messages/messages_en.properties

```diff
@@ -93,7 +93,9 @@ invalidPasswordMinLowerCaseCharsMessage=Invalid password: must contain at least
 invalidPasswordMinDigitsMessage=Invalid password: must contain at least {0} numerical digits.
 invalidPasswordMinUpperCaseCharsMessage=Invalid password: must contain at least {0} upper case characters.
 invalidPasswordMinSpecialCharsMessage=Invalid password: must contain at least {0} special characters.
-invalidPasswordNotUsernameMessage=Invalid password\: must not be equal to the username.
+invalidPasswordNotUsernameMessage=Invalid password: must not be equal to the username.
+invalidPasswordRegexPatternMessage=Invalid password: fails to match regex pattern(s).
+invalidExpiredPasswordMessage=Invalid password: must not be an expired password.
```

Comment:
I don't think this is a clear message to the user. I'd change it to:

Invalid password: must not be equal to any of last {0} passwords